natgeo/modules-analytics
# [MAINT-546] Changing _link to _trackEvent to avoid redirection

Jira Tickets
-----------------
https://jira.nationalgeographic.com/browse/MAINT-543
https://jira.nationalgeographic.com/browse/MAINT-546

The Problem
------------------
Some links were tracked by analytics using the _link function that causes a redirect. This resulted in trouble for links with target="_blank"

The Solution
-----------------
Change the _link function for the _trackEvent function and add a label to differentiate this trackings.

How to Test
-----------------
Go to the main page of the app
Open the network tab
Verify that analytics.min.js is loaded
Verify that analytics is still tracking events in the page (scroll, engagement, dwell, bounce, etc).
